src/bootstrap/builder.rs

```diff
@@ -705,6 +705,68 @@ impl<'a> Builder<'a> {
     ) -> Command {
         let mut cargo = Command::new(&self.initial_cargo);
         let out_dir = self.stage_out(compiler, mode);
+
+        let mut my_out = match cmd {
```

Comment:
I had thought of unifying it but looks like this (`out_dir`) is used lower down https://github.com/rust-lang/rust/pull/52036/files#diff-4bc384d35e57dc577b9378d4d733d7c3R771 and is always the same no matter the passed `cmd` yet `my_out` should be different for each passed `cmd`. Maybe I'm missing something here.

Reply:
Ah, right -- yes, that's correct. I think we'll eventually want to resolve that to be the same (since presumably that's ultimately what we want) but for now it should be enough to just add the comment -- basically saying that this is a command-specific path, unlike out_dir which is broader in scope.